samples/SkiaJvmSample/src/main/kotlin/SkiaJvmSample/App.kt

```diff
@@ -14,7 +14,10 @@ import javax.imageio.ImageIO
 fun main() {
     val windows = 1
```

Comment:
Shalln't we make `windows` property as well?

Reply:
I made via [main(args: Array<String>)]